roles/ansible_tower/tasks/main.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- name: install some useful packages
+  yum: name={{ item }}
+  with_items:
+  - git
+  - python-pip
+  - unzip
+  - python-psphere
+  - ansible
+  - telnet
+  - ack
+
+- name: download Tower setup
+#  get_url: url=http://releases.ansible.com/ansible-tower/setup/ansible-tower-setup-{{tower_version}}.tar.gz
+  get_url: url=http://releases.ansible.com/ansible-tower/setup/ansible-tower-setup-2.1.1.tar.gz
+    dest=/opt/ force=yes
+
+- name: extract Tower
+  unarchive: src=/opt/ansible-tower-setup-2.1.1.tar.gz dest=/opt copy=no
+
+#- name: copy tower_setup_conf.yml
+#  copy: src=tower_setup_conf.yml dest=/opt/ansible-tower-setup-2.1.1 owner=root group=root mode=0644
+#
+#- name: copy inventory
+#  copy: src=inventory dest=/opt/ansible-tower-setup-2.1.1 owner=root group=root mode=0644
+
+- name: run the Tower installer
+  command: chdir=/opt/ansible-tower-setup-2.1.1 creates=/etc/awx/settings.py ./setup.sh
```

Comment:
don't do this, let's do this manually. Again, we don't want this to run on every single config. We also don't want this to hit the DB. Super scary.